app/process/helper/TimeUtils.lua

```diff
@@ -0,0 +1,12 @@
+-- return the number of days that passed since a given timestamp in unix format
```

Comment:
```suggestion
-- return the number of days that passed since a given timestamp in unix format
-- currently everything >736 is considered outdated
```